### PR TITLE
Add 68 landmark functions

### PR DESCRIPTION
**Pull request by harisreedhar**

### CODE REVIEW

File: facefusion/face_analyser.py [hunk 1]
```diff
@@ -52,6 +52,11 @@
 	{
 		'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models/gender_age.onnx',
 		'path': resolve_relative_path('../.assets/models/gender_age.onnx')
+	},
+	'landmark_68':
```
Comment (henryruhs) [inline facefusion/face_analyser.py:56]: face_landmark_68

File: facefusion/face_analyser.py [hunk 2]
```diff
@@ -317,7 +327,8 @@ def get_average_face(frames : List[VisionFrame], position : int = 0) -> Optional
 			embedding = numpy.mean(embedding_list, axis = 0),
 			normed_embedding = numpy.mean(normed_embedding_list, axis = 0),
 			gender = faces[0].gender,
-			age = faces[0].age
+			age = faces[0].age,
```
Comment (henryruhs) [inline facefusion/face_analyser.py:330]:
```
average_face = Face(
			bbox = faces[0].bbox,
			# deprecated
			kps = faces[0].kps,
			landmark = faces[0].landmark,
			score = faces[0].score,
			embedding = numpy.mean(embedding_list, axis = 0),
			normed_embedding = numpy.mean(normed_embedding_list, axis = 0),
			gender = faces[0].gender,
			age = faces[0].age		
```

File: facefusion/face_analyser.py [hunk 3]
```diff
@@ -400,3 +411,19 @@ def filter_by_gender(faces : List[Face], gender : FaceAnalyserGender) -> List[Fa
 		if categorize_gender(face.gender) == gender:
 			filter_faces.append(face)
 	return filter_faces
+
+
+def detect_landmark_68(frame : VisionFrame, bbox : Bbox) -> numpy.ndarray[Any, Any]:
```
Comment (henryruhs) [inline facefusion/face_analyser.py:416]: def detect_face_landmark_68(frame : VisionFrame, bbox : Bbox) -> FaceLandmark68

File: facefusion/typing.py [hunk 4]
```diff
@@ -6,6 +6,10 @@
 Kps = numpy.ndarray[Any, Any]
 Score = float
 Embedding = numpy.ndarray[Any, Any]
+Landmark = TypedDict('Landmark',
```
Comment (henryruhs) [inline facefusion/typing.py:9]: FaceLandmarkSet = TypedDict('FaceLandmarkSet',
{   
	'5' : FaceLandmark68
	'68' : FaceLandmark68
})

File: facefusion/typing.py [hunk 5]
```diff
@@ -14,7 +18,8 @@
 	'embedding',
 	'normed_embedding',
 	'gender',
-	'age'
+	'age',
+	'landmark'
```
Comment (henryruhs) [inline facefusion/typing.py:22]: landmark after bbox

File: facefusion/face_analyser.py [hunk 6]
```diff
@@ -99,7 +106,8 @@ def pre_check() -> bool:
 			MODELS.get('face_detector_yunet').get('url'),
 			MODELS.get('face_recognizer_arcface_inswapper').get('url'),
 			MODELS.get('face_recognizer_arcface_simswap').get('url'),
-			MODELS.get('gender_age').get('url')
+			MODELS.get('gender_age').get('url'),
+			MODELS.get('landmark_68').get('url'),
```
Comment (henryruhs) [inline facefusion/face_analyser.py:110]: landmark before gender_age (order like it is processed)

File: facefusion/face_analyser.py [hunk 7]
```diff
@@ -243,14 +251,16 @@ def create_faces(frame : VisionFrame, bbox_list : List[Bbox], kps_list : List[Kp
 			score = score_list[index]
 			embedding, normed_embedding = calc_embedding(frame, kps)
 			gender, age = detect_gender_age(frame, bbox)
+			landmark : Landmark = {'68' : detect_landmark_68(frame, bbox)}
```
Comment (henryruhs) [inline facefusion/face_analyser.py:254]:
```
bbox = bbox_list[index]
# kps is deprecated
kps = kps_list[index]
landmark : Landmark =\
{
    '5' : kps_list[index]
    '68' : detect_landmark_68(frame, bbox)
}
score = score_list[index]
````

**Review by henryruhs — CHANGES_REQUESTED** [review]
there is an potential false usage of multiple landmarks being applied to one face

File: facefusion/face_analyser.py (hunk 3 above)
Comment (harisreedhar) [reply facefusion/face_analyser.py:416]: It will only return one face landmark depends on face bbox provided.

File: facefusion/typing.py (hunk 4 above)
Comment (harisreedhar) [reply facefusion/typing.py:9]: you mean `'5' : FaceLandmark5`?

**Comment by harisreedhar** [comment]
> there is an potential false usage of multiple landmarks being applied to one face

where is that happening?

File: facefusion/typing.py (hunk 4 above)
Comment (henryruhs) [reply facefusion/typing.py:9]: yes

**Review by henryruhs — APPROVED** [review]

**Pull request merged**